### PR TITLE
po/exposure next - exposure mapping + new collapsible section

### PR DESCRIPTION
**Pull request by TurboGit**
Moved Aurélien branch to the new collapsible section support.

### CODE REVIEW

File: src/iop/exposure.c [hunk 1]
```diff
@@ -559,6 +575,23 @@ void gui_update(struct dt_iop_module_t *self)
...
 
+  g->spot_RGB[0] = 0.f;
+  g->spot_RGB[1] = 0.f;
+  g->spot_RGB[2] = 0.f;
+  g->spot_RGB[3] = 0.f;
+
+  // get the saved params
+  dt_iop_gui_enter_critical_section(self);
+
+  float lightness = 50.f;
```
Comment (TurboGit) [inline src/iop/exposure.c:592]: Maybe I'm missing something, but this does break the reset of the module. That is, even when clicking on reset on the module the lightness spot is not reset to 50%. Is that expected?

Comment (aurelienpierre) [reply src/iop/exposure.c:592]: The reset option applies only on module's params. These are not params but read/written from/to darktablerc, so the reset option does nothing here. The only way to reset is double click on sliders.

**Review by aurelienpierre — APPROVED** [review]

File: src/iop/exposure.c (hunk 1 above)
Comment (TurboGit) [reply src/iop/exposure.c:592]: Indeed ! Thanks.

File: src/iop/exposure.c [hunk 2]
```diff
@@ -552,16 +573,28 @@ void gui_update(struct dt_iop_module_t *self)
...
+  if(dt_conf_key_exists("darkroom/modules/exposure/lightness"))
+    lightness = dt_conf_get_float("darkroom/modules/exposure/lightness");
```
Comment (dterrahe) [inline src/iop/exposure.c:594]: @TurboGit @aurelienpierre Looking at this mechanically while rebasing #11257 and wondering why this isn't using `darktableconfig.xml.in` and setting the default there?

Comment (TurboGit) [reply src/iop/exposure.c:594]: We keep the latest value set by the user, which is another option. The 50.f above could be the default (and it is) but this is used only the first time the module is run, so not very important.

Comment (dterrahe) [reply src/iop/exposure.c:594]: Understood. Is this different from many other variables? The dt_conf infrastructure can deal with defaults _and_ can be used to document the meaning of variables in darktablerc (which now, for these cases, contains mystery variables). I like consistency and copy/paste will always increase entropy, but as you say, it is not very important.

Comment (TurboGit) [reply src/iop/exposure.c:594]: I see, I missed the point, here you go #11279

**Pull request merged**